packages/inter-protocol/src/provisionPoolKit.js

```diff
@@ -21,9 +21,15 @@ import {
 import { InstanceHandleShape } from '@agoric/zoe/src/typeGuards.js';
 import { E } from '@endo/far';
 import { Far } from '@endo/marshal';
+import { isUpgradeDisconnection } from '@agoric/internal/src/upgrade-api.js';
 
 const { details: X, quote: q, Fail } = assert;
 
+const trace = makeTracer('ProvPool');
+
+const FIRST_CAP_ASCII = /^[A-Z][a-zA-Z0-9_$]*$/;
+const FIRST_LOWER_ASCII = /^[a-z][a-zA-Z0-9_$]*$/;
```

Comment:
these are more restricted than ASCII. consider,
```suggestion
const FIRST_CAP_KEYWORD = /^[A-Z]/;
const FIRST_LOWER_KEYWORD = /^[a-z]/;
```

Reply:
I'm changing the names to `FIRST_UPPER_KEYWORD` and `FIRST_LOWER_NEAR_KEYWORD`, but keeping the patterns the same. If it's not a keyword pattern, I'd rather take the `else` fallback.